### DISCUSSION TITLE
orientation matrix-precipitate

### DISCUSSION THREAD

**Discussion by Elimpie** [mtex-toolbox/mtex#1046] [Ask Anything]
Good Morning,
I would like to khow how starting from the euler angles of the Aluminium matrix I can get the euler angles of the precipitate by respecting the orientation conditions shown in the attached image.
I suppose I would need to go through the writing of an orientation matrix….but it is not clear to me  how to write it respecting the given conditions. Moreover, in this case both Aluminium and precipitate have cubic structure what if the precipitate is hexagonal or monoclinic?

Thanks for your help
Elisa
![orientation Al-precipitate](https://user-images.githubusercontent.com/82203900/118769964-08088480-b881-11eb-820b-f491300f6d61.jpg)

**Comment by nyyssont** [comment]
Please look at the documentation regarding misorientations:

https://mtex-toolbox.github.io/MisorientationTheory.html

The bottom has a section on phase transitions. A different structure should not be a problem, as long as you have a suitable crystal symmetry definition for it.

After you have defined some misorientations, I would recommend reading about how to generate phase transition variants:

https://mtex-toolbox.github.io/ParentChildVariants.html

I hope this will get you started.
